src/App.jsx

```diff
@@ -1,107 +1,20 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  comments: commentsFromServer.filter(comment => comment.postId === post.id),
+  user: usersFromServer.find(user => user.id === post.userId),
```

Comment:
will be good to move `commentsFromServer.filter(...)` and `usersFromServer.filter(...)` into separated function